### ISSUE TITLE
Run generator and push changes to generated project repos on publish

### ISSUE THREAD

**Issue by chiubaka** [chiubaka/generator-chiubaka-typescript-package#45]
When the generator is published, we should publish to NPM as you would expect.

However, we should _also_ run the generator with a standard set of options, output that to a submodule, then commit and push updates to that submodule with a message matching the current version of the generator.

#44 likely needs to be completed before this can happen. Standardized configurations for generated packages can be stored as YAML and consumed by the non-interactive mode of the generator.

**Comment by chiubaka** [comment]
I think the case for doing this _now_ is that it sets me up to start easily managing a larger number of generated project types since likely the next two tasks I need to knock out are:
1. At least one other project type
2. Composing project types into monorepos (#127)

Task 1 will help me understand how to start pulling out high-level generators that share care code and remain extremely well-tested.

**Comment by chiubaka** [comment]
Ok, this is working! I think it's OK to close this. One last piece that isn't totally automated: once the submodule has been updated, I'm having trouble getting the root project to auto-commit that submodule update and push that. Going to make a separate issue for that, though.
